frontend/public/style/_layout.scss

```diff
@@ -32,6 +32,13 @@ body,
   flex-direction: column;
 }
 
+.co-m-app__content {
+  display: flex;
+  flex: 1 0 auto;
+  flex-direction: column;
+  height: 100% // doesn't work on safari without height 
```

Comment:
Definitely a solution... tested and it works on Safari 14.

Alternative would be to just set the child to a `flex-grow: 1` to fill the page instead of this height.

Reply:
Did you test the solution in both desktop and mobile? I've encountered issues with the PF drawer on mobile.
I don't know which solution is best without further investigating myself.
So @andrewballantyne are you wanting this change to be made or not? Since you didn't give lgtm